page/alat-berat.php

```diff
@@ -46,56 +47,21 @@
     <div class="row">
         <div class="col-md-3">
             <input type="text" class="search" name="search" placeholder="Quick Search">
-            <ul class="fa-ul list-aside my-3">
-                <li class="py-1"><a href="#">All</a></li>
-                <li class="py-1"><a href="#">Machinery Services</a></li>
-                <li class="py-1"><a href="#">Machinery Equipment</a></li>
-                <li class="py-1"><a href="#">Electric Machines</a></li>
-                <li class="py-1"><a href="#">Agricultural Equipment</a></li>
-                <li class="py-1"><a href="#">Heavy Equipment</a></li>
-            </ul>
         </div>
         <div class="col-md-9 row">
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
-            <div class="col-md-3 mb-4">
-                <div class="card">
-                    <img class="card-img-top" src="img/senjata/SPR_4.jpeg">
-                    <div class="card-body font-weight-bold">
-                        <p>SPR-4</p>
-                    </div>
-                </div>
-            </div>
+            <?php
+                $select = mysqli_query($connection, "select * from produk where jenis_produk='Alat Berat'");
+                while($data = mysqli_fetch_array($select)){
+                    echo "<div class='col-md-3 mb-4'>";
+                    echo "<div class='card'>";
+                    echo "<img class='card-img-top' src='admin/upload/".explode("-", $data["gambar"])[0]."/gambar/".explode("-", $data["gambar"])[1]."'>";
+                    echo "<div class='card-body font-weight-bold'>";
+                    echo "<p>".$data["nama_produk"]."</p>";
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$data["nama_produk"]'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)